test/src/org/labkey/test/tests/targetedms/TargetedMSDocumentFormatsTest.java

```diff
@@ -171,4 +181,17 @@ private void validateCrossLinked(String file, int countWithAnyIon, String modifi
             Assert.assertEquals("Wrong Mz for transition " + index, transitionMz[index++], (Double)row.getValue("Mz"), 0.0001);
         }
     }
+
+    private int countMissingChromatograms(String file) throws Exception
+    {
+        Connection cn = createDefaultConnection();
+
+        List<Filter> filters = List.of(
```

Comment:
Here's an example of firing a SQL query in a test:

https://github.com/labkey/targetedms/blob/develop/test/src/org/labkey/test/tests/targetedms/AbstractQuantificationTest.java#L444

It would be more efficient, but we're not loading a ton of data in this test so I'm not sure it would be noticeable.